charts/victoria-metrics-k8s-stack/templates/_helpers.tpl

```diff
@@ -128,8 +128,12 @@ VMAlert spec
 VM Agent remoteWrite
 */}}
 {{- define "victoria-metrics-k8s-stack.vmAgentRemoteWrite" -}}
-remoteWrite:
-    - url: {{ include "victoria-metrics-k8s-stack.vmInsertEndpoint" . }}/api/v1/write
+{{- $addressList := list (dict "url" (printf "%s/%s" (include "victoria-metrics-k8s-stack.vmInsertEndpoint" . ) "api/v1/write")) -}}
```

Comment:
as far i understand remoteWrite is an array of https://docs.victoriametrics.com/operator/api.html#vmagentremotewritespec 
so we need to add properties from spec to values, we can just merge object